get.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require "connect.php";
+
+function get($db) {
```

Comment:
I'd suggest renaming this function as just get isn't very semantic, even just getBirds would be better